com.github.camel-tooling.lsp.eclipse.client.tests.integration/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: com.github.camel-tooling.eclipse.client.tests.integration;singleton:=true
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 1.0.0
```

Comment:
is that really needed to change? thought it is handled automatically.

Reply:
yes, see https://github.com/camel-tooling/camel-lsp-client-eclipse/pull/43/files#r200651887